### PR TITLE
Fixes and additions to discharge wizard

### PR DESCRIPTION
**Pull request by bshyong**
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5817
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5857
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5843
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5882
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5843
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5855
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5844
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5817
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5849
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5848
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5841
- resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5850

### CODE REVIEW

File: content/pages/request-discharge-updates-and-corrections.md [hunk 1]
```diff
@@ -77,11 +77,40 @@ display_title: Request Discharge Updates and Corrections
...
+                    <button class="usa-button-unstyled usa-accordion-button" aria-controls="dbq4" itemprop="name">Can I apply for VA benefits in the meantime?</button>
+                    <div id="dbq4" class="usa-accordion-content" itemprop="acceptedAnswer" itemscope itemtype="http://schema.org/Answer">
+                      <div itemprop="text">
+                        <div class="va-alert usa-alert usa-alert-warning">
+                          <div class="va-alert-body usa-alert-body">
+                            <div class="usa-alert-text usa-alert-heading">
+                              <p>Even with a less than honorable discharge, you may be able to still access some VA benefits through the <a href="https://www.benefits.va.gov/BENEFITS/docs/COD_Factsheet.pdf">Character of Discharge or Character of Service Determination process.</a></p>
+                            </div>
+                          </div>
+                          <div class="cf"></div>
```
Comment (U-DON) [inline content/pages/request-discharge-updates-and-corrections.md]: What's the `cf` for?

File: src/js/discharge-wizard/containers/GuidancePage.jsx [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
+import { branchOfService, board, formData, venueAddress } from '../utils';
 import { venueWarning } from '../config';
 
 class GuidancePage extends React.Component {
+  componentDidMount() {
+    const script = document.createElement('script');
+    script.type = 'text/javascript';
+    script.async = true;
+    script.src = 'https://standards.usa.gov/assets/js/vendor/uswds.min.js';
+    document.getElementsByTagName('head')[0].appendChild(script);
```
Comment (U-DON) [inline src/js/discharge-wizard/containers/GuidancePage.jsx]: Why does this page specifically need this?

**Review by U-DON — APPROVED** [review]
LGTM.

File: src/js/discharge-wizard/containers/GuidancePage.jsx (hunk 2 above)
Comment (bshyong) [reply src/js/discharge-wizard/containers/GuidancePage.jsx]: @U-DON the USWDS accordion requires this script :(

There may be a better way to do this but haven't explored it too much in depth

File: content/pages/request-discharge-updates-and-corrections.md (hunk 1 above)
Comment (bshyong) [reply content/pages/request-discharge-updates-and-corrections.md]: good catch; will remove this

File: src/js/discharge-wizard/containers/GuidancePage.jsx (hunk 2 above)
Comment (U-DON) [reply src/js/discharge-wizard/containers/GuidancePage.jsx]: Huh. Then wouldn't tons of other pages that use the accordion pattern need this? Why isn't this just part of our includes?

Comment (bshyong) [reply src/js/discharge-wizard/containers/GuidancePage.jsx]: Maybe - I was planning to explore further when I had time. The other accordions are all on static pages and people drop the script tag into the bottom of markdown file. Seems like uswds does something onload to initialize the accordions so I am getting it to work here by injecting it

**Pull request merged**